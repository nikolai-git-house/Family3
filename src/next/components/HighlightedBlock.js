import React from 'react'

import Frow from './Frow'
import Theme from '../styles/theme'

export default ({ children, ...props }) => (
  <Frow container>
    <section {...props} className="highlighted">
      <Frow xs="8/10" md="8/10" hug>{children}</Frow>
      <style jsx>{`
        .highlighted {
          background-color: ${ Theme.colors.accentSecondary };
          font-family: 'Noto Serif', serif;
          /* line-height: 1.48; */
          line-height: 1.66667;
          /* padding: ${ Theme.xs.vr(2) } 3rem; */
          padding: ${ Theme.xs.vr(2) } 0;
          margin-bottom: ${ Theme.xs.vr(1) };
        }

        @media (min-width: 992px) {
          .highlighted {
            padding: ${ Theme.md.vr(2) } 0;
            margin-bottom: ${ Theme.md.vr(1) };
          }
        }
      `}</style>
    </section>
  </Frow>
)
