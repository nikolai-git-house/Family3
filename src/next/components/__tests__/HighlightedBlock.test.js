import React from 'react'
import renderer from 'react-test-renderer'


import HighlightedBlock from '../HighlightedBlock'


describe("HighlightedBlock", () => {
  it("renders", () => {
    const actual = renderer.create(<HighlightedBlock>test</HighlightedBlock>)
    const actualTree = actual.toJSON()

    expect(actualTree).toMatchSnapshot()
  })
})
