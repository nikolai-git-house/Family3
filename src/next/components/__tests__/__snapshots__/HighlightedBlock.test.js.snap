// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HighlightedBlock renders 1`] = `
<section
  className="frow jsx-2877306380 highlighted"
  style={
    Object {
      "flexGrow": undefined,
      "flexShrink": undefined,
    }
  }
>
  <div
    className="col-xs-8-10 col-md-8-10"
    style={
      Object {
        "flexGrow": undefined,
        "flexShrink": undefined,
      }
    }
  >
    test
  </div>
</section>
`;
